www-src/app/change/change_dispatcher.coffee

```diff
@@ -0,0 +1,76 @@
+ChangeFileHandler = require "./change_file_handler"
+ChangeEventHandler = require "./change_event_handler"
+ChangeContactHandler = require "./change_contact_handler"
+ChangeTagHandler = require "./change_tag_handler"
+log = require('../lib/persistent_log')
+    prefix: "ChangeDispatcher"
+    date: true
+
+###*
+ * ChangeDispatcher allows to launch the good manager with specific state
+ *
+ * @class ChangeDispatcher
+###
+module.exports = class ChangeDispatcher
+
```

Comment:
We could simplify a lot this file using a dict :
```
@changeHadlers = 
    file: new ChangeFileHandler config
    event: new ChangeEventHandler
   ....

```
Dispatch would be two lines, and isDispatched one ; )